commands/xinfo-stream.md

```diff
@@ -107,12 +107,13 @@ OK
        14) 1) 1) "name"
               2) "Alice"
               3) "seen-time"
-              4) (integer) 1638125153423
-              5) "pel-count"
-              6) (integer) 1
-              7) "pending"
-              8) 1) 1) "1638125133432-0"
-                    2) (integer) 1638125153423
-                    3) (integer) 1
->
+              4) (integer) 1638125133422
+              5) "active-time"
+              6) (integer) 1638125133432
```

Comment:
so many of the consumer related fields are undocumented in this sub-command?
where do users get the info about them?

Reply:
I believe these are currently simply not documented.

Reply:
XINFO STREAM FULL is not documented at all

Reply:
https://github.com/redis/redis-doc/issues/2230